ingress/controllers/gce/controller/utils.go

```diff
@@ -34,7 +34,10 @@ import (
 	"github.com/golang/glog"
 )
 
-const allowHTTPKey = "kubernetes.io/ingress.allowHTTP"
+const (
+	allowHTTPKey    = "kubernetes.io/ingress.allow-http"
```

Comment:
I just used the pattern we use for static pods in the kubelet: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/pod_update.go#L26. 

Allow-http is shared but global-static-ip-name is gce specific. Easy enough to rename since this isn't in a relased image yet.